packages/clerk-js/src/ui/components/UserButton/useMultisessionActions.tsx

```diff
@@ -33,10 +33,10 @@ export const useMultisessionActions = (opts: UseMultisessionActionsParams) => {
     );
   };
 
-  const handleManageAccountClicked: React.MouseEventHandler<HTMLButtonElement> = () => {
+  const handleManageAccountClicked = () => {
     if (opts.userProfileMode === 'navigation') {
       return navigate(opts.userProfileUrl).finally(async () => {
-        await sleep(1000);
+        await sleep(300);
```

Comment:
❓ So, the final change is to reduce this sleep timer? Why?

Reply:
Checking whether this is a external or internal navigation is too complicated for this simple scenario. We might revise in the future, but this should be enough for now